plugins/modem-manager/fu-plugin-modem-manager.c

```diff
@@ -14,7 +14,8 @@
 #include "fu-mm-device.h"
 
 struct FuPluginData {
-	MMManager	*manager;
+	MMManager *manager;
```

Comment:
If we *use* the manager before MM is active, what happens? Does it auto-activate then? If not, do we need to check `manager_ready` before we call the various things on manager?

Reply:
You won't call anything on the manager as long as there are no modems reported, so there is no need to check manager_ready for any operation really. If there are modems reported, manager will be ready implicitly.

The gboolean is really really only used to properly log "ModemManager no longer unavailable" when we have lost the access to the service. If we don't want that specific logging, we could totally remove the flag I'd say (and just call `g_signal_handlers_disconnect_by_func()` unconditionally in `fu_plugin_mm_teardown_manager()`. Can do that if you prefer to avoid the gboolean.

Reply:
No, that's fine. Your logic works. :)